cibuildwheel/linux.py

```diff
@@ -130,6 +130,26 @@ def build(options: BuildOptions) -> None:
 
             call(['docker', 'start', container_name])
 
+            if options.before_all:
+                call(
+                    ['docker', 'exec', '-i', container_name] + shell_cmd,
+                    universal_newlines=True,
+                    input='''
+                        PS4='    + '
+
+                        set -o errexit
+                        set -o xtrace
+
+                        {environment_exports}
```

Comment:
Just had an idea re. available python, maybe i was overthinking this. it's always there on the docker images anyway - why not just do this?

```suggestion
                        export PATH=/opt/python/cp38-cp38:$PATH

                        {environment_exports}
```

Am I missing an obvious downside?

Reply:
Hm. I do not see any. @YannickJadoul ?

Reply:
Yeah, this could work.

Advantage: a decent Python version available.
Disadvantages that I now see: we need to update it with every Python release, and we can't just document that you get the plain docker image without any changes.